src/wp-includes/functions.php

```diff
@@ -8757,7 +8759,8 @@ function recurse_dirsize( $directory, $exclude = null, $max_execution_time = nul
  */
 function clean_dirsize_cache( $path ) {
 	if ( ! is_string( $path ) || empty( $path ) ) {
-		trigger_error(
+		wp_trigger_error(
+			__FUNCTION__,
```

Comment:
The function's name is already incorporated into the translated version of the error message.

```suggestion
			'',
```